### PR TITLE
Fixing Xplat create config command

### PR DESCRIPTION
**Pull request by nbikhchandani**
The content to be added to Changelog is as follows:
* Adding options to Xplat CLI command config create
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nbikhchandani__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nabikhch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js [hunk 1]
```diff
@@ -446,7 +449,103 @@ describe('arm', function() {
         });
       });
 
-      it('delete should delete hdinsight linux cluster', function(done) {
+      it('config create should create a Azure HDinsight cluster configuration', function(done) {
+		this.timeout(hdinsightTest.timeoutLarge);
+		var cmd = util.format('hdinsight config create %s --json', configFile).split(' ');
+		suite.execute(cmd, function(result) {
+			result.exitStatus.should.equal(0);
```
Comment (amarzavery) [inline test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js]: @nbikhchandani - indentation 2 spaces.

File: test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js [hunk 2]
```diff
@@ -469,21 +472,23 @@ describe('arm', function() {
...
-		});
-		var cmd = util.format('hdinsight config create %s --overwrite overwrite --json', configFile).split(' ');
-		suite.execute(cmd, function(result) {
-			result.exitStatus.should.equal(0);
-			fs.stat(configFile, function(err, stat2) {
-				assert.notEqual(stat2.mtime.getTime(), beforetimestamp);
-			});
-		    if (!suite.isPlayback()) {
-				setTimeout(function () {
+
```
Comment (amarzavery) [inline test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js]: indent 2 spaces

File: test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js [hunk 3]
```diff
@@ -446,7 +452,105 @@ describe('arm', function() {
...
+		this.timeout(hdinsightTest.timeoutLarge);
+		var beforetimestamp;
+		fs.stat(configFile, function(err, stat1) {
+			beforetimestamp = stat1.mtime.getTime();
+
+			var cmd = util.format('hdinsight config create %s --overwrite overwrite --json', configFile).split(' ');
+			suite.execute(cmd, function(result) {
+				result.exitStatus.should.equal(0);
+				fs.stat(configFile, function(err, stat2) {
+					assert.notEqual(stat2.mtime.getTime(), beforetimestamp);
```
Comment (amarzavery) [inline test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js]: @nbikhchandani - https://travis-ci.org/Azure/azure-xplat-cli/jobs/126233683
The CI is failing on this assertion
```
  1) arm hdinsight cluster config create overwrite should overwrite if config file already exists:
     Uncaught AssertionError: 1461794746000 != 1461794746000
      at /home/travis/build/Azure/azure-xplat-cli/test/commands/arm/hdinsight/arm.hdinsight-cluster-crud-tests.js:480:13
      at Object.oncomplete (fs.js:108:15)
```

**Pull request merged**